README.md

```diff
@@ -111,6 +111,14 @@ Copy the appropriate `![Name](link)` element and paste it in your Markdown file
 | TravisCI       | ![TravisCI](https://img.shields.io/badge/travisci-%232B2F33.svg?style=for-the-badge&logo=travis&logoColor=white)                   | `![TravisCI](https://img.shields.io/badge/travisci-%232B2F33.svg?style=for-the-badge&logo=travis&logoColor=white)`                   |
 
 
+[![](https://img.shields.io/badge/back%20to%20top-%E2%86%A9-blue)](#shortcuts)
+
+### CD
```

Comment:
Thanks for this addition.
Kindly add this new section to the [Shortcuts](https://github.com/Ileriayo/markdown-badges/blob/4e2d7df4303e89929b69f7475b01d10e74a26f15/README.md#shortcuts)

Reply:
Have added a shortcut. 👍